client/my-sites/email/controller.js

```diff
@@ -101,9 +101,11 @@ export default {
 				source={ pageContext.query.source }
 			/>
 		) : (
-			<EmailProvidersComparisonStacked
+			<EmailProvidersStackedComparison
 				comparisonContext="email-purchase"
 				selectedDomainName={ pageContext.params.domain }
+				selectedEmailProviderSlug={ pageContext.query.provider }
+				selectedIntervalLength={ pageContext.query.interval }
```

Comment:
I think we should be enforcing the correct type for this value in this code, though it may be better to default to `null` if we get a bad or otherwise invalid value, so the defaulting can occur within the component.

Reply:
I'll do that 👍.